CMSgov/easi-app
# Noref/contract numbers add dataloaders

# EASI-3204, NOREF


## Changes and Description

- The point of this PR is solely to get dataloaders attached to the `dump` and `submit` cobra commands. If not, they will panic
- Add context and dataloaders to dump commands. Clean up some error messages to be more descriptive



## How to test this change

- run both `dump` and `submit` commands and confirm everything works as expected



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
